test/format/CONFIG.disagg

```diff
@@ -8,9 +8,10 @@ precise_checkpoint=1
 disagg.page_log=palm
 disagg.enabled=1
 disagg.layered=1
+runs.rows=1000000:3000000
```

Comment:
We should do this at the evergreen level and make short running tasks, with limiting rows ops and tables there. At the end of it all, all those short running tasks will go away and this file should randomise to run anything. Second reason is that for now follower mode can handle larger runs and we do not want to limit that.

What do you think?

Reply:
Sure I will move to evergreen level, i noticed CONFIG.stress only have `runs.rows=1000000:5000000`, we should consider increasing there too